diff -Nur a/Makefile b/Makefile
--- a/Makefile	2015-11-09 18:47:51.000000000 +0000
+++ b/Makefile	2016-03-26 23:37:05.230500590 +0000
@@ -6,7 +6,7 @@
 #EXTRA_CFLAGS += -Werror
 #EXTRA_CFLAGS += -pedantic
 #EXTRA_CFLAGS += -Wshadow -Wpointer-arith -Wcast-qual -Wstrict-prototypes -Wmissing-prototypes
-EXTRA_CFLAGS += -Werror=incompatible-pointer-types
+#EXTRA_CFLAGS += -Werror=incompatible-pointer-types
 
 EXTRA_CFLAGS += -Wno-unused-variable
 EXTRA_CFLAGS += -Wno-unused-value
